python/paddle/fluid/distribute_transpiler.py

```diff
@@ -812,6 +824,9 @@ def _get_lr_ops(self):
                 find_ops.append(op)
         # make a union find struct by the ops in default_main_program
         ufind = UnionFind(block.ops)
+
+        # TODO(panyx0718): If lr_ops connects with other training
```

Comment:
We'll use SSA to replace the current ufind structure. For current deeplearning programs, it works fine.

Reply:
Removed this comment